concent_api/core/tests/utils.py

```diff
@@ -807,27 +807,6 @@ def _get_deserialized_force_get_task_result_failed(  # pylint: disable=no-self-u
                 task_to_compute=task_to_compute,
             )
 
-    def _store_golem_messages_in_database(
```

Comment:
This function was not used anywhere?

Reply:
Nope, I remembered there was a function like that, because I wanted to use it initially, but apparently it's broken/outdated/unused.